firebase-functions/src/main/java/com/google/firebase/functions/HttpsCallOptions.kt

```diff
@@ -45,30 +38,32 @@ class HttpsCallOptions {
    * @param timeout The length of the timeout, in the given units.
    * @param units The units for the specified timeout.
    */
-  void setTimeout(long timeout, TimeUnit units) {
-    this.timeout = timeout;
-    this.timeoutUnits = units;
+  fun setTimeout(timeout: Long, units: TimeUnit) {
+    this.timeout = timeout
+    timeoutUnits = units
   }
 
   /**
    * Returns the timeout for calls from this instance of Functions.
    *
    * @return The timeout, in milliseconds.
    */
-  long getTimeout() {
-    return timeoutUnits.toMillis(timeout);
+  fun getTimeout(): Long {
+    return timeoutUnits.toMillis(timeout)
   }
 
-  boolean getLimitedUseAppCheckTokens() {
-    return limitedUseAppCheckTokens;
+  /** Creates a new OkHttpClient with these options applied to it.  */
+  fun apply(client: OkHttpClient): OkHttpClient {
+    return client
+            .newBuilder()
+            .callTimeout(timeout, timeoutUnits)
+            .readTimeout(timeout, timeoutUnits)
+            .build()
   }
 
-  /** Creates a new OkHttpClient with these options applied to it. */
-  OkHttpClient apply(OkHttpClient client) {
-    return client
-        .newBuilder()
-        .callTimeout(timeout, timeoutUnits)
-        .readTimeout(timeout, timeoutUnits)
-        .build();
+  companion object {
+    // The default timeout to use for all calls.
+    private const val DEFAULT_TIMEOUT: Long = 70
+    private val DEFAULT_TIMEOUT_UNITS = TimeUnit.SECONDS
```

Comment:
Why did `DEFAULT_TIMEOUT_UNITS` not get converted to a const when `DEFAULT_TIMEOUT` was?

Reply:
Because Kotlin only allows primitives and strings to be `const`